tests/backup/backup_helper.go

```diff
@@ -1969,9 +1969,11 @@ func ValidateBackup(ctx context.Context, backupName string, orgID string, schedu
 
 	if len(errStrings) > 0 {
 		return fmt.Errorf("ValidateBackup Errors: {%s}", strings.Join(errStrings, "}\n{"))
-	} else {
-		return nil
 	}
+
+	err = validateCRCleanup(backupName, theBackup.Cluster, theBackup.OrgId, theBackup.Namespaces, ctx, "backup")
```

Comment:
Do we need to validate anything on application clusters as well in addition to backup application cluster ?

Reply:
We are validating the restore CR cleanup on application/restore clusters